hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapService.java

```diff
@@ -338,4 +339,16 @@ public void rollbackTransaction(String transactionId) {
     public void dispatchEvent(EventData event, EntryListener listener) {
         dispatcher.dispatchEvent(event, listener);
     }
+
+    @Override
+    public Map<String, LocalMultiMapStats> getStats() {
+        Map<String, LocalMultiMapStats> mapStats = new HashMap<String, LocalMultiMapStats>();
+        for (int i = 0; i < nodeEngine.getPartitionService().getPartitionCount(); i++) {
```

Comment:
It's sad that we have very different statistics handling for each service. I had created an issue for this: https://github.com/hazelcast/hazelcast/issues/3703

Calling `createStats` method in `createDistributedObject` is an improvement but doesn't solve the problem of accessing statistics without creating proxies.